### ISSUE TITLE
[FEATURE] Update default task file/readme about default shell

### ISSUE THREAD

**Issue by AvinZarlez** [TobiahZ/processing-vscode#69]
**Is your feature request related to a problem? Please describe.**
Some users are having issues with Windows using Bash. It's likely some kind of file name/directory path issue?

**Describe the solution you'd like**
Ideally, I'd want Processing to run in the shell of the user's choice. But I guess since there is syntax differences between cmd and bash that may be causing an issue?

Some users have fixed themselves. Maybe rather than changing the task file, can update the readme for users to tweak themselves if its an issue?

**Describe alternatives you've considered**
If I had the time I'd try to test out these cases and explore why bash may be failing, and see if I can make the script more robust to work regardless of shell used (rather than forcing it to summon cmd.exe)

**Additional context**
The issue was first raised in #56 by @WissenIstNacht and @micuat

**Comment by AvinZarlez** [comment]
Updated Readme in 1.4

**Comment by micuat** [comment]
after some search, this worked on my Windows 10 / bash (reference: https://github.com/Microsoft/vscode/issues/46567)
both `strong` and `weak` quote works, but `escape` doesn't.

```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "Run Sketch",
      "type": "shell",
      "group": {
        "kind": "build",
        "isDefault": true
      },
      "command": "${config:processing.path}",
      "presentation": {
        "echo": true,
        "reveal": "always",
        "focus": false,
        "panel": "dedicated"
      },
      "args": [
        "--force",
        {
          "value": "--sketch=${workspaceRoot}",
          "quoting": "strong"
        },
        {
          "value": "--output=${workspaceRoot}\\out",
          "quoting": "strong"
        },
        "--run"
      ]
    }
  ]
}
```

**Comment by AvinZarlez** [comment]
Ooh, great find!

I'll test it out and implement it in the next update.

(I'll need to make sure I still respect the Windows args though, because of the slash vs double slash should still be an issue mac vs. windows)

Though they're also the issue of updating the json and educating people to delete it and start over. Hopefully people having this issue will be able to figure that out.

- [x] Implement this in json (changing just windows args)
- [x] Test
- [x] Make clear in update and readme about deleting task json and regenerating. Maybe add in readme a note on extension version number task json was last updated? Maybe add a comment to the json itself?

**Comment by AvinZarlez** [comment]
Implemented in 1.4.1
